apps/chat/src/components/Marketplace/MarketplaceFilterbar.tsx

```diff
@@ -137,47 +149,57 @@ const MarketplaceFilterbar = () => {
         />
         <ActionButton
           isOpen={showFilterbar}
-          onClick={onHomeClick}
+          onClick={() => onTabClick(MarketplaceTabs.HOME)}
```

Comment:
Here useCallback doesn't make sense because you are passing callback as prop anyway. I think we should create `handleHomeTabClick` and `handleMyAppsTabClick` that call handleTabClick internally and pass the args `MarketplaceTabs.HOME` and `MarketplaceTabs.MY_APPLICATIONS`